WebContent/js/components/dialogs/datasets/CreateMemberDialog.js

```diff
@@ -33,12 +35,22 @@ export default class CreateMemberDialog extends React.Component {
         this.setState({
             memberName: newValue,
         });
+        // disable the Submit, when PDS member name has length more than 8
+        const regex = /^([A-Z#@$][A-Z0-9#@$-]{0,7})/g;
```

Comment:
Strange but true that a PDS member can actually have a name that is any 8 bytes, invisible or not. However, most APIs don't accept this, so, the regex you have here is probably correct for the API you use.